index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Documentation Page</title>
+</head>
+<body>
+  <div class="container">
+    <div class="sideNav">
+      <h1>JS documentation</h1>
+      <ul>
+        <li><a href="#intro">Introduction</a></li>
+        <li><a href="#"></a></li>
+        <li><a href="#what">What you should already know</a></li>
+        <li><a href="#js">Javascript or Java</a></li>
+        <li><a href="#hello">Hello world</a></li>
+        <li><a href="#variable">Variable</a></li>
+        <li><a href="#declare">Declaring variable</a></li>
+        <li><a href="#scope">Variable scope</a></li>
+        <li><a href="#global">Global variables</a></li>
+        <li><a href="#constant">Constants</a></li>
+        <li><a href="#data">Data types</a></li>
+        <li><a href="#condition">if...else statement</a></li>
+        <li><a href="#function">Function declarations</a></li>
+        <li><a href="#refrence">Refrence</a></li>
+      </ul>
+    </div>
+    <!-- end of side navigation -->
+    <div class="main">

```

Comment:
- [ ] I suggest you explain why this change is required, that will make it very easy to implement the changes because the one who will read it will know the context

Reply:
Please use the main tag instead of a div because:
* Specifies the main content of a document